cmd/services.rb

```diff
@@ -15,10 +15,10 @@ def services_args
         If `sudo` is passed, operate on `/Library/LaunchDaemons` (started at boot).
         Otherwise, operate on `~/Library/LaunchAgents` (started at login).
 
-        [`sudo`] `brew services` [`list`]:
+        [`sudo`] `brew services` [`list`] (`--json`):
         List all managed services for the current user (or root).
 
-        [`sudo`] `brew services info` (<formula>|`--all`):
+        [`sudo`] `brew services info` (<formula>|`--all`|`--json`):
         List all managed services for the current user (or root).
```

Comment:
I know this wasn't changed by you, but it could be better still
```suggestion
        List full info on selected services.
```

Reply:
```suggestion
        List information about all managed services for the current user (or root).
```

Don't like the `full info`, personally.